src/elements/content-uploader/__tests__/ItemList.test.js

```diff
@@ -1,13 +1,29 @@
 import * as React from 'react';
 import { mount } from 'enzyme';
+// TODO Providers can be removed when converted to RTL - providers are included in the testing utility library
+import { IntlProvider } from 'react-intl';
+import { TooltipProvider } from '@box/blueprint-web';
```

Comment:
How come this test isn't migrated, but `ItemAction.test.tsx` is? Did we decide to do this as a separate task?